src/modules/menu/components/header/HeaderLoadingBar.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div class="header-loading-bar"></div>
+</template>
+
+<script>
+export default {}
+</script>
+
+<style lang="scss">
+@import 'node_modules/bootstrap/scss/bootstrap';
+@import 'src/scss/variables';
+$loading_element_width: 40rem;
```

Comment:
Why here using snake case ? and not kebab case like the others; eg. `$header-height` ?

Reply:
my bad, I'll switch to `the-correct-notation-for-css`